src/testoptions.c

```diff
@@ -395,6 +395,9 @@ int initialize_tests(Connection *ctl, TestOptions *options, char *buff,
   if (useropt & TEST_META) {
     add_test_to_suite(&first, buff, buff_strlen, TEST_META);
   }
+  // Meta data that should be unconditionally saved is set here.
```

Comment:
Yesterday you were telling me how there are four different options that can be enabled or disabled (I don't remember all of them right now). Are any of the others helpful to have here? Please add them if so.